contracts/zero-ex/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "0.20.0-bsc",
```

Comment:
So this is not gonna work with our current publish job. Immediate TODO is to update the job/scripts to allow for publishing named forks. The alternative is to gitpkg everything (yuck).